### PR TITLE
[Merged by Bors] - sync code cleanup

### PR DESCRIPTION
**Pull request by almogdepaz**
## Motivation
code cleanup

## Changes
cosmetic

## Test Plan
same

### CODE REVIEW

File: sync/block_listener.go [hunk 1]
```diff
@@ -10,9 +10,10 @@ import (
 	"time"
 )
 
+//BlockListener Listens to blocks propagated in gossip
```
Comment (y0sher) [inline sync/block_listener.go:13]: I believe in order for all the tools like godoc to work with the comments you need a single space between the comment token and the comment itself.

```suggestion
// BlockListener Listens to blocks propagated in gossip
```

File: sync/block_listener_test.go [hunk 2]
```diff
@@ -34,9 +34,9 @@ func (pm PeersMock) Close() {
 	return
 }
 
-func ListenerFactory(serv service.Service, peers Peers, name string, layer types.LayerID) *BlockListener {
+func ListenerFactory(serv service.Service, peers peers, name string, layer types.LayerID) *BlockListener {
```
Comment (y0sher) [inline sync/block_listener_test.go]: needs comment

File: sync/fetch_queue.go [hunk 3]
```diff
@@ -9,29 +9,29 @@ import (
...
 }
 
 type fetchJob struct {
-	items []Item
+	items []item
 	ids   []types.Hash32
 }
 
 //todo make the queue generic
 type fetchQueue struct {
```
Comment (y0sher) [inline sync/fetch_queue.go:27]: I guess some commenting about how `fetchQueue` works can be useful even though it's not exported

**Review by y0sher — APPROVED** [review]

File: sync/block_listener.go (hunk 1 above)
Comment (almogdepaz) [reply sync/block_listener.go:13]: no this works, i checked

File: sync/fetch_queue.go (hunk 3 above)
Comment (almogdepaz) [reply sync/fetch_queue.go:27]: true, im gonna do more doc pr's after this one

File: sync/block_listener.go (hunk 1 above)
Comment (almogdepaz) [reply sync/block_listener.go:13]: what's the convention? space?

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Comment by almogdepaz** [comment]
bors merge

**Pull request closed without merging**